### PR TITLE
Third Party Examples

### LINKED ISSUE #1515: Add page for Vendor Example Projects
Suggestion: Adding a page under Examples / Tutorials that links to example projects for all vendors that have a vendor dependency and have examples easily accessible.

REV: https://docs.revrobotics.com/sparkmax/software-resources/spark-max-code-examples

CTRE: https://github.com/CrossTheRoadElec/Phoenix-Examples-Languages

PhotonVision: https://docs.photonvision.org/en/latest/docs/examples/index.html

Kauai Labs: https://pdocs.kauailabs.com/navx-mxp/examples/

Don't know if this is allowed or if this preferences certain vendors over others, but if vendors are free to PR their own links onto this page, it should be fair in my eyes.

### PR DESCRIPTION
**Pull request by jasondaming**
Closes #1515

### CODE REVIEW

**Comment by sciencewhiz** [comment]
I'm thinking it makes more sense to integrate into the 3rd party libraries article rather then having a separate examples article that someone has to discover.

File: source/docs/software/examples-tutorials/third-party-examples.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+============================
+
+This list helps you find example programs for use with third party devices.  You can find support for many of these third parties on the :ref:`docs/software/support/support-resources:Support Resources` page.
+
+* `Cross The Road Electronics (CTRE) <https://github.com/CrossTheRoadElec/Phoenix-Examples-Languages>`__
+* `REV Robotics <https://github.com/REVrobotics/SPARK-MAX-Examples>`__
+* `Limelight <https://docs.limelightvision.io/en/latest/cs_drive_to_goal_2019.html>`__ (more case studies can be found on the left)
+* `PhotonVision <https://docs.photonvision.org/en/latest/docs/examples/index.html>`__
+* `Kauai Labs (navX) <https://pdocs.kauailabs.com/navx-mxp/examples/>`__
+* `Swerve Drive Specialties (SDS) <https://github.com/SwerveDriveSpecialties/swerve-template>`__
```
Comment (Daltz333) [inline source/docs/software/examples-tutorials/third-party-examples.rst:11]: Should we sort alphabetically?

File: source/docs/software/examples-tutorials/third-party-examples.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Third Party Example Projects
+============================
+
+This list helps you find example programs for use with third party devices.  You can find support for many of these third parties on the :ref:`docs/software/support/support-resources:Support Resources` page.
+
+* `Cross The Road Electronics (CTRE) <https://github.com/CrossTheRoadElec/Phoenix-Examples-Languages>`__
+* `REV Robotics <https://github.com/REVrobotics/SPARK-MAX-Examples>`__
+* `Limelight <https://docs.limelightvision.io/en/latest/cs_drive_to_goal_2019.html>`__ (more case studies can be found on the left)
```
Comment (Daltz333) [inline source/docs/software/examples-tutorials/third-party-examples.rst:8]:
```suggestion
* `Limelight <https://docs.limelightvision.io/en/latest/cs_drive_to_goal_2019.html>`__ (more case studies can be found on the left)
```

As a reader, I don't know what "more" is really referring to as there is no noun that it was previously describing. Also the reader has no idea what a "case study" is. I'd either omit this entire sentence or rephrase to something close to "example code can be found in the left menu in the form of case studies", but that is quite lengthy.

**Review by Daltz333 — CHANGES_REQUESTED** [review]

File: source/docs/software/examples-tutorials/third-party-examples.rst (hunk 1 above)
Comment (jasondaming) [reply source/docs/software/examples-tutorials/third-party-examples.rst]: done

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**